src/oaklib/interfaces/differ_interface.py

```diff
@@ -275,34 +276,34 @@ def diff(
             if mappings_added_set:
                 for mapping in mappings_added_set:
                     predicate, xref = mapping
-                    edge_created = EdgeCreation(
+                    mapping_created = MappingCreation(
                         id=_gen_id(),
                         subject=entity,
                         predicate=predicate,
                         object=xref,
                     )
-                    yield edge_created
+                    yield mapping_created
             if mappings_removed_set:
                 for mapping in mappings_removed_set:
                     predicate, xref = mapping
-                    deleted_edge = EdgeDeletion(
+                    deleted_mapping = MappingDeletion(
```

Comment:
@cmungall should `MappingDeletion` be [RemoveMapping](https://incatools.github.io/kgcl/RemoveMapping/)?

Reply:
Thank is correct!